doc/FAQ.md

```diff
@@ -0,0 +1,104 @@
+determine how many tasks have been generated. To check this look at the UI for your spark
+job and see how many tasks are being run. In the current Shell a small progress bar is shown
+when running stages, the numbers represent (Completed Tasks + Running Tasks) / Total Tasks 
+
+    [Stage 2:=============================================>  (121 + 1) / 200]0
+
+If you see that only a single task has been created this means that the Cassandra Token range has
+not been split into a enough tasks to be well parallelized on your cluster. The number of 
+Spark partitions(tasks) created is directly controlled by the setting `spark.cassandra.input.split.size`.
+This number reflects the approximate number of live Cassandra Partitions in a given Spark partition.
+To increase the number of Spark Partitions decrease this number from the default (100k) to one that
+will sufficiently break up your C* token range. This can also be adjusted on a per cassandraTable basis
+with the function `withReadConf` and specifying a new `ReadConf` object.
+
+If there are more than one tasks but only a single machine is working make sure that the job itself
+has been allocated multiple executor slots to work with. This is set at the time of SparkContext
+creation with `spark.cores.max` in the `SparkConf` and cannot be changed during the job. 
+
+### Why can't the spark job find Spark Cassandra Connector Classes? (ClassNotFound Exceptions for SCC Classes)
+
+The most common cause for this is that the executor classpath does not contain the Spark Cassandra Connector
+jars. The simplest way to add these to the class path is to use SparkSubmit with the --jars option pointing
+to your Spark Cassandra Connector assembly jar. If this is impossible for some reason the second best option
+is to manually distribute the jar to all of your executors and add the jar's location to the executor classPath
+in the SparkConf or spark-defaults.conf.
+
+### Where should I set configuration options for the connector?
+
+The suggested location is to use the `spark-defaults.conf` file in your spark/conf directory but 
+this file is ONLY used by spark-submit. Any applications not running through spark submit will ignore
+this file. 
+ 
+For those you should set the options in the `SparkConf` object used to 
+create your `SparkContext`. Usually this will take the form of a series of statements that look like
+
+    val conf = SparkConf()
+      .set("Option","Value")
+      ...
+      
+    val sc = SparkContext(conf)
+
+### Why are my write tasks timing out/ failing?
+
+The most common cause of this is that Spark is able to issue write requests much more quickly than
+Cassandra can handle them. This can lead to GC issues and build up of hints. If this is the case
+with your application try lowering the number of concurrent writes and the current batch size using
+the following options.
+
+   spark.cassandra.output.batch.size.rows
+   spark.cassandra.output.concurrent.writes
```

Comment:
You should probably mention the `output.throughput_mb_per_sec` option available since 1.2.